File: VB/EFCore/ViewSettingsEF/ViewSettingsEF.Module/BusinessObjects/SettingsStore.vb
Imports System.ComponentModel
Imports DevExpress.Persistent.BaseImpl.EF

Namespace ViewSettingsEF.Module.BusinessObjects

    Public Class SettingsStore
        Inherits BaseObject

        <Browsable(False)>
        Public Overridable Property Xml As String

        Public Overridable Property Name As String

        <Browsable(False)>
        Public Overridable Property OwnerId As String

        Public Overridable Property IsShared As Boolean

        <Browsable(False)>
        Public Overridable Property ViewId As String
    End Class
End Namespace

File: CS/EFCore/ViewSettingsEF/ViewSettingsEF.Module/BusinessObjects/SettingsStore.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using DevExpress.Persistent.BaseImpl;
using DevExpress.ExpressApp;
using System.ComponentModel;
using DevExpress.Persistent.BaseImpl.EF;

namespace ViewSettingsEF.Module.BusinessObjects {
    public class SettingsStore : BaseObject {
        [Browsable(false)]
        public virtual string Xml { get; set; }
        public virtual string Name { get; set; }
        [Browsable(false)]
        public virtual string OwnerId { get; set; }
        public virtual Boolean IsShared { get; set; }
        [Browsable(false)]
        public virtual string ViewId { get; set; }
    }
}
